# Smithsonian Peer-led Bioinformatics series
Bioinformatics training series led by members of the Smithsonian community

##2016 Series
Talks are on Thursdays from 2-3pm in WG33, Natural History Building.

Date | Speaker | Topic
-----|---------|------
3/17/2016 | Andy Gottscho | PyRAD - a python pipeline for Restriction-Associated-DNA sequencing
3/24/2016 | Michael Trizna | Jupyter Notebook for workflow management
3/31/2016 | Paul Frandsen | Data partitioning
4/7/2016 | Rebecca Dikow | Genome Assembly
4/14/2016 | Vanessa Gonzalez | Gene Prediction (Augustus)
4/21/2016 | Carolyn Teplot | Analysis of coexpression patterns in genes

##2015 Series
Date | Speaker | Topic | Slides 
-----|---------|-------|--------
4/20/2015 | Whole genome alignment/phylogeny | Rebecca Dikow | 
4/27/2015 | submitting to cluster with qsub, scripting (may be 2 hrs) | Paul Frandsen | 
5/4/2015 | genotype and snp calling | HC Lim | 
5/11/2015 | gene prediction and annotation using Trinotate | Cheryl Lewis Ames | 
5/18/2015 | sequence capture experiment design | Missy Hawkins | 
6/1/2015 | metabarcoding data OTU calling, community-level analysis | Matt Leray | 
6/8/2015 | Phylogenomic analysis | Ehsan Kayal | 
6/15/2015 | Tricking out your Mac: Installing analysis software on your Mac | Matt Kweskin | 
6/22/2015 | Web Apollo & Gene Ontology Resources for Functional Annotations | Monica C. Munoz-Torres | 
6/29/2015 | RAD-seq library prep (pros/cons), some snp calling tools, population genomics analysis | Tammy Wilbert
